app/views/accounts/_phone.html.erb

```diff
@@ -3,17 +3,15 @@
     <h2 class="grid-col-fill margin-y-0 padding-right-2">
       <%= t('account.index.phone') %>
     </h2>
-    <div class="grid-col-auto">
-      <% if EmailPolicy.new(current_user).can_add_email? %>
-        <%= link_to(
-              prefix_with_plus(t('account.index.phone_add')),
-              add_phone_path,
-              class: 'account-action-button',
-            ) %>
+  </div>
+  <div class="grid-row margin-bottom-1 margin-top-0">
+    <div class="grid-col">
+      <a id="phones"></a>
+      <% if flash[:phone_error] %>
+        <%= render partial: "layouts/alert_error", locals: { message: flash[:phone_error] } %>
```

Comment:
Can we use the AlertComponent here?

Reply:
If we use alert it will display at the top. But we want phone number to have its own. As our next ticket we are also going to do the same for emails.

Reply:
@aduth is more familiar so may have some better guidance, but something like the following should be able to render it in the same location:

```ruby
  <%= render AlertComponent.new(type: 'error', message: flash[:phone_error]) %>
```

Reply:
`AlertComponent` is just a simple view component, and doesn't control where it's displayed. I think what @mitchellhenke is suggesting is that we could replace this with:

```suggestion
        <%= render AlertComponent.new(type: :error) { flash[:phone_error] } %>
```

Reply:
Ohh cool I was hoping we already had something. I'll look into that. It looks much cleaner.